app/component/menu/menu-list/menu-list.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+require('./_menu-list.scss');
+
+module.exports = {
+  template: require('./menu-list.html'),
+  controller: ['$log', '$location', 'bizService', 'picService', menuListController],
+  controllerAs: 'menuListCtrl',
+  bindings:{
+    biz: '<'
+  }
+};
+
+function menuListController($log, $location, bizService, picService) {
+  $log.debug('menuListController');
+
+  this.updatePics = function(biz) {
+    console.log('updatePics');
+    bizService.getPics(biz._id)
+    .then( biz => {
+      console.log(biz);
+      this.pics = biz.menuPics.map( menuPic => {
+        return menuPic.imageURI;
+      });
+    });
+    // .then( biz => {
```

Comment:
Unless you intend to bring any of the commented code back later, it's cleaner to remove before pushing.